File: Team1/Riya Tanwar/find-pivot-index.java
class Solution {
    public int pivotIndex(int[] nums) {
    int sum=0,left_sum=0;
    for(int i:nums){
    sum+=i;
    }
    for(int i=0;i<nums.length;i++){
    sum-=nums[i];
    if(sum==left_sum){
    return i;
    }
    left_sum+=nums[i];
    }
    return -1;
    }
    }

File: Team2/ShrutiSinha/Week1_Shruti/FindPivotIndex.cpp
class Solution {
public:
    int pivotIndex(vector<int>& nums) {
        int n=nums.size();
        int sum=0, leftsum=0;
        for(int i=0;i<n;i++){
            sum+=nums[i];
        }
        for(int i=0;i<n;i++){
            if(leftsum==sum-leftsum-nums[i])
                return i;
            leftsum+=nums[i];
        }
        return -1;
    }
};